Mathlib/Order/WellFoundedSet.lean

```diff
@@ -5,17 +5,17 @@ Authors: Aaron Anderson
 -/
 import Mathlib.Data.Prod.Lex
 import Mathlib.Data.Sigma.Lex
-import Mathlib.Order.Antichain
-import Mathlib.Order.OrderIsoNat
-import Mathlib.Order.WellFounded
+import Mathlib.Order.RelIso.Set
+import Mathlib.Order.WellQuasiOrder
 import Mathlib.Tactic.TFAE
 
 /-!
 # Well-founded sets
 
-A well-founded subset of an ordered type is one on which the relation `<` is well-founded.
+This file introduces specializations of `WellFounded` and `WellQuasiOrdered` for sets.
```

Comment:
To me these are generalisations, since we're only requiring the conditions to hold in a restricted context. Can we compromise on "introduces versions of"?